ledger-core/v2/virtual/object/awaittablefill.go

```diff
@@ -0,0 +1,40 @@
+package object
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/injector"
+)
+
+type SMAwaitTableFill struct {
```

Comment:
@sexyhamster Looks like the same as SMAwaitDelegate. Maybe we can reuse that already have?

Reply:
Why we need a new SM?